l10n.toml

```diff
@@ -0,0 +1,87 @@
+
+basepath = "."
+
+locales = [
```

Comment:
We actually only ship to maybe 10 system locales and don't include an in-app locale switcher – the remaining locales are leftovers from the Focus fork. Should we reduce this list to avoid unnecessary translations? If so, I can talk to Ashley about the locales we ship to. fwiw, I'm not sure the locales that FFTV ships to are the same as FFES.

I realize now that we may have already been overtranslating on FFTV 😞

Reply:
I've sync'ed with Delphine on the list, update coming.

Reply:
Thanks! :) @Pike Can we consider this list of locales to be the defacto supported locale list on FFES?

Reply:
@Delphine wanted to circle back with @athomasmoz on that, but it's likely close.

@Delphine, can you file issues on both FFTV and FFES to remove localized files that are not part of our supported list of locales?

Reply:
@Pike Yes, will do :)